src/metrics/index.md

```diff
@@ -0,0 +1,53 @@
+# Standard Metrics
+
```

Comment:
I suggest adding an intro to help the reader get their bearings.

```suggestion
 This section provides an overview of standard metrics used at Mozilla. 
 Here you'll find the definitions and descriptions for each.
 
 For a deep dive into these metrics, see [the GUD documentation](../tools/gud.md).
```

Reply:
It looks like there are also more detailed definitions later in the docs under "Telemetry Behaviour Reference", might be worth a mention of that as well.